proto/dydxprotocol/vault/share.proto

```diff
@@ -21,3 +21,22 @@ message OwnerShare {
   string owner = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
   NumShares shares = 2 [ (gogoproto.nullable) = false ];
 }
+
+// LockedShares stores for an owner their total number of locked shares
+// and a schedule of share unlockings.
+message LockedShares {
+  // Address of the owner of below shares.
+  string owner_address = 1;
+  // Total number of locked shares.
```

Comment:
nit: Thought through this a bit more, I think we can remove this property, and have this be a function on the `LockedShares` type, that iterates through `unlock_details` and returns the total.
Once you have a pointer to the type, all the unlock details are unmarshalled anyways, so I don't think we save anything on serde by having this aggregated value in `LockedShares`, and the remaining iteration / arithmetic will be fast.

Reply:
Agreed that it's an extra thing to keep in mind to update both and it's simpler to iterate through to get total locked shares